README.md

```diff
@@ -1 +1,196 @@
-# todo-list-review
\ No newline at end of file
+# to-do-list
+
+
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
```

Comment:
Kindly remove this line. It clutters your file and makes it look unprofessional.